embedchain/embedchain.py

```diff
@@ -224,7 +224,7 @@ def chat(self, input_query, config: ChatConfig = None):
             context,
             chat_history=chat_history,
         )
-        answer = self.get_answer_from_llm(prompt)
+        answer = self.get_answer_from_llm(prompt, config)
         memory.chat_memory.add_user_message(input_query)
```

Comment:
does that mean that streaming won't work with chat?

Reply:
it will

Reply:
sorry for the oversight.